src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -1006,10 +1024,7 @@ class npc_meteor_strike_initial : public CreatureScript
                         me->SetOrientation(angle[i]);
                         Position newPos = me->GetNearPosition(10.0f, 0.0f); // Exact distance
                         if (Creature* meteor = me->SummonCreature(NPC_METEOR_STRIKE_NORTH + i, newPos, TEMPSUMMON_TIMED_DESPAWN, 30000))
-                        {
-                            meteor->SetOrientation(angle[i]);
```

Comment:
Don't ask me why, i don't understood the code, it was basicaly trial and error, but look at the results, shouldn't be like this?
http://prntscr.com/bkkl0s
http://prntscr.com/bkkkl7
http://prntscr.com/bkkkda
http://prntscr.com/bkkkl7
That `static_cast<float>(M_PI / 5.0f), static_cast<float>(M_PI / 2.0f)` makes this happen, the fire shapes between a cross and a X shape